test/resources/kubeconfig.resource

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Library    OperatingSystem
+Library    SSHLibrary
+Library    String
+
+*** Keywords ***
+Get Kubeconfig
+    [Documentation]    Get the kubeconfig file from the host argument and return contents
+    [Arguments]    ${host}
+    SSHLibrary.Open Connection    ${host}
+    SSHLibrary.Login              ${USHIFT_USER}    allow_agent=True
```

Comment:
Can we avoid the requirement to setup ssh agent by using the following key word instead:
SSHLibrary.Login With Public Key      ${USHIFT_USER}  ${SSH_PRIV_KEY}

I'd like to avoid the hassle of setting up additional processes on test drivers such as the Jenkins agent containers used by the QE pipeline.